.github/workflows/build.yml

```diff
@@ -43,10 +43,49 @@ jobs:
       if: "env.PUSH_VERSION_COMMIT != ''"
       run: git push origin ${{ github.event.ref }}
 
+  build_conda:
+    needs: prepare
+    runs-on: "ubuntu-latest"
+    defaults:
+      run:
+        shell: bash -l {0}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+            auto-update-conda: true
+            miniforge-variant: Mambaforge
+            channels: conda-forge
+            python-version: '3.10'
+            activate-environment: build
+            environment-file: conda-env.yml
+            use-mamba: true
+
+      - name: Build Unix Standalone Executable
+        run: |
+            sudo apt-get -y install zip
+            conda activate build
+            python devscripts/update-version.py ${{ needs.prepare.outputs.version_suffix }}
+            python devscripts/make_lazy_extractors.py
+            python pyinst.py --onedir
+            (cd ./dist/yt-dlp_linux && zip -r ../yt-dlp_linux.zip .)
+            python pyinst.py
+
+      - name: Get SHA2-SUMS
+        id: get_sha
+        run: |
+
```

Comment:
This is a mistake in the build file. Pls remove

Reply:
Ok. I removed it from both `build_conda` and `build_unix`.